pwnlib/fmtstr.py

```diff
@@ -151,33 +174,36 @@ def fmtstr_payload(offset, writes, numbwritten=0, write_size='byte'):
     if write_size not in ['byte', 'short', 'int']:
         log.error("write_size must be 'byte', 'short' or 'int'")
 
+    # number: number of write_size in a word
+    # step: amount of bytes in write_size
+    # mask: mask for written values
+    # formatz: format size modifier for %n
+    # declage: amount of bits in write_size
     number, step, mask, formatz, decalage = config[context.bits][write_size]
```

Comment:
Thanks for adding comments here! The existing code was a bit hard to follow without these